TechDebtsTracking/core/src/main/java/com/novoda/support/annotations/TechDebt.java

```diff
@@ -0,0 +1,32 @@
+package com.novoda.support.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.SOURCE)
+@Target({ElementType.ANNOTATION_TYPE,
+        ElementType.CONSTRUCTOR,
+        ElementType.FIELD,
+        ElementType.LOCAL_VARIABLE,
+        ElementType.METHOD,
+        ElementType.PACKAGE,
+        ElementType.PARAMETER,
+        ElementType.TYPE})
+public @interface TechDebt {
+
+    enum Size {
+        XSMALL,
```

Comment:
I like the idea of having an indication about the size.
I would keep it simple though, with just SMALL, MEDIUM and LARGE for a rough estimation.
It's true that having more options is just giving more flexibility, so no problem 👍